### PR TITLE
[#174011616] Blog > Multiple authors not showing properly

### PR DESCRIPTION
**Pull request by nykka**
Pivotal Story reference:

This PR:

- Shows only one category list on mutiple authors as noticed by @ABizzinotto and @cleicar on https://github.com/ombulabs/fastruby.io/pull/212

Screen Shot 2020-08-10 at 14 07 00

### CODE REVIEW

**Comment by ABizzinotto** [comment]
@rdormer when you have a minute could you help review this PR as well? Thanks!

File: _includes/author.html [hunk 1]
```diff
@@ -1,71 +1,85 @@
...
-      nopin="nopin"
-    />
-    {%- endif -%}
-  </div>
+  <div class="author-block">
+    <div class="author-photo">
+      {%- if author.gravatar -%}
+      <img
+        class="author-thumb"
+        src="//2.gravatar.com/avatar/{{ author.gravatar }}?s=50"
```
Comment (rdormer) [inline _includes/author.html:7]: should this be http:// ?

Comment (nykka) [reply _includes/author.html:7]: I don't know, it was like that before

Comment (nykka) [reply _includes/author.html:7]: Do you think I should change it, @rdormer?

**Comment by ABizzinotto** [comment]
@FionaDL this looks good too

**Pull request merged**